### PR TITLE
Added support for user choosing bulk app translation download format

### PR DESCRIPTION
**Pull request by orangejenny**
##### SUMMARY
https://dimagi-dev.atlassian.net/browse/ICDS-1140

##### FEATURE FLAG
Bulk multimedia path management

##### PRODUCT DESCRIPTION
Adds option to select either single or multi sheet format for download. Previously, selecting all languages always downloaded the multi-sheet format, and selecting a single language always downloaded the single-sheet format. 

This does block multi-language, single-tab download because it would be a very large file and no one has asked for it. ICDS's request is to be able to download the multi-sheet format for a single language.

Screen Shot 2020-01-02 at 8 27 49 PM

### CODE REVIEW

File: corehq/apps/app_manager/templates/app_manager/partials/bulk_upload_app_translations.html [hunk 1]
```diff
@@ -17,15 +17,72 @@
...
     {% endif %}
   {% endif %}
 {% endblock %}
+
+{% block download_file %}
+  <p>
+    <a id="download_link" href="{{ bulk_upload.download_url }}" data-bind="attr: {href: url}">
+      <button class="btn btn-primary" data-bind="disable: disableDownload">
+        <i class="fa fa-cloud-download"></i>
+        {% blocktrans with plural_noun=bulk_upload.plural_noun %}
```
Comment (mkangia) [inline corehq/apps/app_manager/templates/app_manager/partials/bulk_upload_app_translations.html]: this is interesting, does `plural_noun` change based on selection?

**Review by mkangia — APPROVED** [review]
This is great!
I am sure users would love this. Always like when things are explicit, that langs logic was something.

Just wanted to check if you cross checked if this needs modification to where we validate app translations?

File: corehq/apps/app_manager/templates/app_manager/partials/bulk_upload_app_translations.html (hunk 1 above)
Comment (mkangia) [reply corehq/apps/app_manager/templates/app_manager/partials/bulk_upload_app_translations.html]: ah i see, its just set in context.

**Comment by orangejenny** [comment]
> Just wanted to check if you cross checked if this needs modification to where we validate app translations?

Good point. Happily, it looks like validation is set up to handle both the single and multi sheet formats: https://github.com/dimagi/commcare-hq/blob/4f83a4a8eb1002d9e9e163fd297e0f81ea476348/corehq/apps/translations/validator.py#L55

**Comment by mkangia** [comment]
> it looks like validation is set up to handle both the single and multi sheet formats

Well done Manish. Hope that still works

**Comment by orangejenny** [comment]
🤞

**Pull request merged**